src/footer/css/footer-breadcrumb.pcss

```diff
@@ -0,0 +1,82 @@
+/* Global Footer Breadcrumb */
+/* ========================================================================== */
+
+.esri-footer-section--0 {
+
+	&.hidden{
+		display: none;
+	}
+
+	@media (--gfoot-s) {
+		
+	}
+
+	@media (--gfoot-m) {
+		
+	}
+
+	@media (--gfoot-l) {
+		padding-left: calc(50% - 1424px / 2);
+    padding-right: calc(50% - 1424px / 2);
+	}
+
+}
+
+.esri-footer-breadcrumb {
+	box-sizing: border-box;
+	padding: 25px 0;
+}
+
+.clear {
```

Comment:
This class name is far too generic, you will run into collisions on lots of pages